components/Tabs/examples/supplemental/TabBarIconExampleSupplemental.m

```diff
@@ -22,33 +22,215 @@
 
 #import "TabBarIconExampleSupplemental.h"
 
+#import "MaterialAppBar.h"
 #import "MaterialButtons.h"
 #import "MaterialTabs.h"
+@import MaterialComponents.MaterialPalettes;
 
 @implementation TabBarIconExample (Supplemental)
 
 - (void)setupExampleViews {
+  [self setupAppBar];
+
+  [self setupScrollView];
+  [self setupScrollingContent];
+
+  [self setupAlignmentButton];
+}
+
+- (void)setupAlignmentButton {
+  self.alignmentButton = [[MDCRaisedButton alloc] init];
```

Comment:
Refer to `_alignmentButton` instead here to reduce the possibility of side effects?